main.py

```diff
@@ -1,105 +1,16 @@
 import sqlite3
+from datab_funct import *
```

Comment:
```suggestion
import datab_funct
```

Reply:
też tak miałem na początku, ale jest błąd: 
"Traceback (most recent call last):
  File "C:\Users\mateusz.saganowski\PycharmProjects\pythonProject\main.py", line 118, in <module>
    conn = create_connection(database)
NameError: name 'create_connection' is not defined"

Reply:
Hmm bo pewnie trzeba wywoływać datab_funct.nazwa_funkcji. Dobra zostaw jak jest :P

Reply:
Faktycznie, wtedy by zadziałało, ale to w każdym miejscu, przy każdej funkcji musiałbym dodać ten przedrostek.

Reply:
nom, bez sensu.

Reply:
https://docs.python-guide.org/writing/structure/#modules
z best practices to jednak trzeba zmienić. from coś import * jest chujowe :D

Reply:
Dobrze, przekonwertuję całość.